tests/integration/test_permissions.py

```diff
@@ -58,6 +58,7 @@ def _create_and_index_record(record):
 def sample_record(app):
     record = {
         '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'authors': [],
```

Comment:
@kaplun I left it empty because the validator checks if `authors` key is exist or not so value didn't matter. I can apply a dummy one.